lib/request-wrapper.ts

```diff
@@ -67,42 +68,42 @@ export class RequestWrapper {
     this.axiosInstance = axios.create(axiosConfig);
 
     // set debug interceptors
-    if(process.env.NODE_DEBUG === 'axios') {
+    if(process.env.NODE_DEBUG === 'axios' || process.env.DEBUG) {
       this.axiosInstance.interceptors.request.use(config => {
-        console.debug('Request:');
+        logger.debug('Request:');
         try {
-          console.debug(JSON.stringify(config, null, 2));
+          logger.debug(JSON.stringify(config, null, 2));
         } catch {
-          console.debug(config)
+          logger.error(config)
         }
```

Comment:
There's something still not quite right here.  If someone sets `NODE_DEBUG=axios` but _does not_ set `DEBUG=node-sdk-core:debug`, then they will get _no_ debug output (if I'm reading this correctly).

It appears that we don't actually document the behavior of `NODE_DEBUG=axios`, so I guess we need to decide how we want this to work.  We could just drop support for `NODE_DEBUG=axios` now that we have this new logging support.

Reply:
Ah yes you're right. Was that flag initially added just to log the request and response being intercepted?

Reply:
@mkistler - I removed the log flag from here and I just enable all the loggers if the axios flag is set. Please take a look at ``logger.ts``

Reply:
~@mkistler - I removed the DEBUG flag from here and I just enabled all the loggers if the axios flag is set. Please take a look at `logger.ts`~ I just realized removing ``DEBUG`` flag was a mistake. Since removing made that block dependent on the axios flag being set now. Re-adding it back in.